webpack.config.js

```diff
@@ -75,7 +75,7 @@ module.exports = (env = {}) => {
     output: {
       filename: '[name]-[hash].js',
       path: bundleOutputDir,
-      publicPath: dev ? 'http://127.0.0.1:4000/dist/' : undefined,
+      publicPath: dev ? 'http://127.0.0.1:4000/dist/' : '/static/studio/',
```

Comment:
This is unrelated to the changed location of static files --- it makes images assets loaded with the required work, e.g. image in header
<img width="295" alt="Screen Shot 2020-03-10 at 8 03 09 PM" src="https://user-images.githubusercontent.com/163966/76369636-86c37280-630a-11ea-89f2-fcd7ad9cbbee.png">